docs/team/william.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I think we are supposed to name this file as our github username i.e. wjacobw.md